openapi.yaml

```diff
@@ -0,0 +1,106 @@
+openapi: 3.0.0
+info:
+  version: 1.0.0
+  title: Simple API
+  description: A simple API to illustrate OpenAPI concepts
+
+servers:
+  - url: https://example.io/v1
+
+
+paths:
+  /allFunds:
+    get:
+     description: Returns a list of funds,with filter [from-to] 
+     responses:
+        '200':
+          description: Successfully returned a list of funds [from-to]
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: object
+                  required:
+                    - id
+                  properties:
+                    funds:
+                      type: integer
+                    from:
+                      type: integer
+                    to:
+                      type: integer
+                    
+
+        '400':
+          description: Invalid request
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:   
+                  message:
+                    type: string
+  /addIncome:                
```

Comment:
/addTransaction, and in the request body specify the enum Type of the transaction.